Dataset string memcap 3910/v5 (#11105)
OISF/suricata: doc/userguide/upgrade.rst

  Instead, both the SDP parser and logger depend on being invoked by another parser (or logger).
- ARP decoder and logger have been introduced.
  Since ARP can be quite verbose and produce many events, the logger is disabled by default.

> - Datasets of the type String had a bug fix which takes into account the length of the string
>   as well when checking for memcaps. This might lead to memcaps being hit on older deployments.

Reviewer: Suggestion to make it more assertive:
```suggestion
- Datasets of the type String had a bug fix which now takes into account the length of the string
  as well when checking for memcaps. This may lead to memcaps being hit for older setups that didn't
  take that into account.
```
